modify-publication/src/main/java/no/unit/nva/publication/modify/ModifyPublicationHandler.java

```diff
@@ -19,6 +17,8 @@
 import org.apache.http.HttpStatus;
 import org.slf4j.LoggerFactory;
 
+import static nva.commons.utils.JsonUtils.objectMapper;
```

Comment:
We should decide if we want the static imports to be above or under the non-static imports

Reply:
This is intelliJ Optimize Imports